include/quic/datagram.hpp

```diff
@@ -84,14 +172,30 @@ namespace oxen::quic
         {
             log::debug(log_cat, "{} called", __PRETTY_FUNCTION__);
             size_t sum{0};
-            if (user_buffers.empty())
+            if (send_buffer.empty())
                 return sum;
-            for (const auto& [data, store] : user_buffers)
-                sum += data.size();
+            for (const auto& entry : send_buffer.buf)
+                sum += entry.size();
             return sum;
         };
-        bool has_unsent() const override { return is_empty(); }
+        bool has_unsent() const override { return not is_empty(); }
         void wrote(size_t) override { log::debug(log_cat, "{} called", __PRETTY_FUNCTION__); };
+        std::vector<ngtcp2_vec> pending() override
+        {
+            log::warning(log_cat, "{} called", __PRETTY_FUNCTION__);
+            return std::vector<ngtcp2_vec>{};
```

Comment:
Purely a style thing, but this can be written as `return {};` if you don't feel like repeating the return type again.